crates/astria-bridge-withdrawer/src/metrics.rs

```diff
@@ -44,6 +45,21 @@ impl Metrics {
     pub(crate) fn increment_sequencer_submission_failure_count(&self) {
         self.sequencer_submission_failure_count.increment(1);
     }
+
+    #[expect(
+        clippy::cast_precision_loss,
+        reason = "metric with potential loss of precision, noted with a log warning when it occurs"
+    )]
+    pub(crate) fn set_sequencer_block_settled_value(&self, value: u128) {
+        let max_u32: u128 = u32::MAX.into();
+        if value > max_u32 {
+            tracing::warn!(
+                "sequencer_block_settled_value is being set with a value that exceeds u32::MAX, \
```

Comment:
Just add the value that you are casting here. Might be interesting to look at.

Also use the const name instead of writing it out in the message (`SEQUENCER_BLOCK_...`) in case we rename it. Can be an event field or just a part `{}` in the format args.